mocks/log.go

```diff
@@ -126,47 +126,47 @@ func (msw *MockSyslogWriter) Clear() {
 }
 
 // Close releases resources. No other methods may be called after this.
-func (msw *MockSyslogWriter) Close() error {
```

Comment:
You are correct, sir. In the latest rev, it looks like everything works peachily if I restore the pointer receiver functions to the way they were.

Is it really the case that implementing an interface with functions that have pointer receivers means only pointers to the receiver type are considered to implement the interface? But a non-pointer value of the receiver type doesn't implement the interface? That seems very confusing!